src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -56,6 +56,7 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
+    <a href="/profile">Profile</a>
```

Comment:
Same as prev comment, order of pages in header

Reply:
fixed